frontend/src/libraries/explorer-wamp/transactions.ts

```diff
@@ -435,24 +463,36 @@ export default class TransactionsApi extends ExplorerApi {
           transactionInfo.hash,
           transactionInfo.signerId,
         ]);
+
         transactionInfo.status = Object.keys(
           transactionExtraInfo.status
         )[0] as ExecutionStatus;
 
-        const actions = transactionExtraInfo.transaction.actions;
-        transactionInfo.actions = actions.map((action: RpcAction | string) => {
-          if (typeof action === "string") {
-            return { kind: action, args: {} };
-          } else {
-            const kind = Object.keys(action)[0] as keyof RpcAction;
-            return {
-              kind,
-              args: action[kind],
-            };
+        const actions = transactionExtraInfo.transaction.actions.map(
+          (action: RpcAction | string) => {
+            if (typeof action === "string") {
+              return { kind: action, args: {} };
+            } else {
+              const kind = Object.keys(action)[0] as keyof RpcAction;
+              return {
+                kind,
+                args: action[kind],
+              };
+            }
           }
-        });
-
+        );
+        transactionInfo.actions = actions;
         transactionInfo.receiptsOutcome = transactionExtraInfo.receipts_outcome as ReceiptOutcome[];
+        transactionInfo.receiptsList = {
+          receipts: transactionExtraInfo.receipts as ReceiptOutcomeInfo[],
+          receipts_outcome: transactionExtraInfo.receipts_outcome as ReceiptOutcome[],
```

Comment:
`receiptsList` feels unnecessary (it sticks out immediately to me due to its naming which does not reflect the object that it defines)